deployment/server.js

```diff
@@ -115,6 +114,7 @@ io.on("connection", (socket) => {
     if (room) {
       if (room.room.removeNode) {
         room.room.removeNode(socket, req.socketID, req.room, room.room);
+        socket.to(req.room).emit("decrementPeerCount");
```

Comment:
same as above, might be more secure instead of sending to the whole room.

Reply:
Makes sense. Fixing it